test/build.sh

```diff
@@ -16,8 +16,9 @@ cmakeConfigure()
     local CMAKEARGS=(..
            "-DCMAKE_Fortran_COMPILER=${fcomp}"
            "-DCMAKE_INSTALL_PREFIX=${idir}"
-           "-DCMAKE_CXX_COMPILER=g++"
-           "-DCMAKE_C_COMPILER=gcc"
+           "-DCMAKE_CXX_COMPILER=CC"
+           "-DCMAKE_C_COMPILER=cc"
```

Comment:
As far as I understand, the compiler should always be `cc` for C and `CC` for C++ on Cray machines. The wrapper takes care of compiling with the right compiler and link against the correct version of the MPI libraries and other runtimes.